src/API/WorkController.ts

```diff
@@ -13,6 +13,11 @@ interface WorksFilters {
   readonly limit?: number
 }
 
+interface WorksWithCount {
+  readonly count: number
+  readonly works: ReadonlyArray<string>
```

Comment:
Why `ReadonlyArray<string>`? Aren't this something like `Work & { timestamp: PoetTimestamp }`? 

Might want to create a `WorkWithTimestamp` interface for that. It would be specific to the API module.

Reply:
Honestly not really sure what I was thinking there.